src/Components/test/E2ETest/Tests/EventTest.cs

```diff
@@ -174,10 +174,10 @@ public void DragDrop_CanTrigger()
 
         actions.Perform();
         // drop doesn't seem to trigger in Selenium. But it's sufficient to determine "any" drag event works
-        Browser.Equal("ondragstart,", () => output.Text);
+        Browser.Equal("dragstart,", () => output.Text);
     }
 
-    [Fact(Skip = "https://github.com/dotnet/aspnetcore/issues/32373")]
+    [Fact]
```

Comment:
This may be resolved given the below change (was missing a comma), trying it out on the CI.